site/_posts/2020-09-03-oidc-auth-api-usage.md

```diff
@@ -80,13 +85,39 @@ oidc_client_id="my-oidc-client"
 
 # OIDCOauthIntrospectionEndpoint from the OpenID-Connect configuration
 oidc_auth_introspection_endpoint="https://keycloak.example.com/auth/realms/miq/protocol/openid-connect/token/introspect"
+```
+
+  * The value of **oidc_provider_token_endpoint** varies based on the configuration and the OpenID-Connect Provider being used. 
+
+  * When using IBM's Identiy Access Management (IAM) for the OpenID-Connect Provider, the value for **oidc_provider_token_endpoint** will be the value of OIDCProviderTokenEndpoint from the OpenID-Connect configuration with **token** replace with **identitytoken**.
```

Comment:
Seems a bit hard to follow telling me replace this with that,  maybe it can be something like: 

```
Each OpenID-Connect Provider defines its own endpoints for token generation and introspection

Example for Keycloak:

    OIDCProviderTokenEndpoint ....
    OIDCOAuthIntrospectionEndpoint ....

    oidc_provider_token_endpoint=https://....
    oidc_auth_introspection_endpoint=https://....

Example for IAM:

    OIDCProviderTokenEndpoint  ....
    OIDCOAuthIntrospectionEndpoint  ....

    oidc_provider_token_endpoint=https://....
    oidc_auth_introspection_endpoint=https://....
```

Reply:
Good idea. Will do